servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/XdsOutlierDetector.java

```diff
@@ -67,10 +71,13 @@ final class XdsOutlierDetector<ResolvedAddress, C extends LoadBalancedConnection
     private final SequentialExecutor sequentialExecutor;
     private final Executor executor;
     private final String lbDescription;
+    private final PublisherSource.Processor<Void, Void> eventStreamProcessor =
```

Comment:
consider renaming this to indicate that it is related to the health status changes